entries/b/bankofamerica.com.json

```diff
@@ -4,14 +4,9 @@
       "bofa.com",
       "mbna.com"
     ],
-    "contact": {
-      "twitter": "BofA_Help",
-      "facebook": "BankofAmerica",
-      "form": "https://www.bankofamerica.com/deposits/contact-bank-of-america-faqs/"
-    },
```

Comment:
The contact information is needed until the service supports passkey.
If BofA supports passkeys, you need to not only remove the `contact` element, but also add either the `passwordless` or `mfa` element, depending on what type of passkey support they have.